pysd/py_backend/builder.py

```diff
@@ -138,8 +138,16 @@ def build_element(element, subscript_dict):
                     'ulines': '-' * len(element['real_name']),
                     'contents': contents.replace('\n',
                                                  '\n' + ' ' * indent)})  # indent lines 2 onward
-    element['doc'] = element['doc'].replace('\\', '\n    ')
-
+    
+    element['doc'] = element['doc'].replace('\\', '\n    ').encode('unicode-escape')
```

Comment:
Encode special unicode chars, as Python2 don't support unicode chars in source code files